lib/screen/home/home_navigation_page.dart

```diff
@@ -33,8 +39,13 @@ class _HomeNavigationPageState extends State<HomeNavigationPage> {
       });
       _pageController.jumpToPage(_selectedIndex);
       if (index == 0) {
+        _homePageKey.currentState?.refreshFeeds();
         _homePageKey.currentState?.refreshTokens();
       }
+      if (index == 1) {
+        _homePageKey.currentState?.refreshFeeds();
```

Comment:
I see both if still using `_homePageKey.currentState?.refreshFeeds();`
Why not move it out?